sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1829,6 +1830,64 @@ export class ApplicationService extends RecordDataModelService<Application> {
     });
   }
 
+  /**
+   * Return scholastic standing application
+   * @param locationId location id.
+   * @param applicationId application id.
+   * @returns Application.
+   */
+  async processScholasticStanding(
```

Comment:
As we got an opportunity to separate the getApplication() part, we can re-use this(my best suggestion) to get application details. 
![image](https://github.com/user-attachments/assets/3129baec-0e26-42b9-b0ba-17a038f0d326)